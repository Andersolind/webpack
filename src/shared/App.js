import React from 'react';
const App = () => <div data-testid='hello'>Hello World</div>;

export default App;
