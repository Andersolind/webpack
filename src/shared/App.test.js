import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';
it('should render an empty message when no books', () => {
  const elem = renderer.create(<App />).toJSON();
  expect(elem).toMatchSnapshot();
});
