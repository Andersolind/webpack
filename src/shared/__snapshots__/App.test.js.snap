// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an empty message when no books 1`] = `
<div>
  Hello World
</div>
`;
